src/main/java/org/terasology/caves/CaveFacetProvider.java

```diff
@@ -60,10 +59,11 @@ public void process(GeneratingRegion region) {
         float[][] caveNoiseValues = new float[][]{caveNoise[0].noise(facet.getWorldRegion()),caveNoise[1].noise(facet.getWorldRegion())};
 
         for (Vector3i pos : region.getRegion()) {
+            float frequencyReduction = (float)Math.min(0.3,Math.max(0,(100+pos.y)/400.0)); //0: no reduction, 0.7: pretty much no caves. Also somewhat increases the tendency of caves to loop rather than continuing indefinitely.
```

Comment:
Where do `100` and `400` come from? Just based on experiments with different values ending up with some configuration that "looks good" with the default Perlin world gen (we should to Simplex...)

Reply:
Yes, just experiments with Core. The previous version used the SurfaceHeightFacet to get the depth, but I expect that sudden changes in depth underneath things like the valleys would cause weird artefacts in the shapes of caves beneath them, so I just made it constant instead. Using the SeaLevelFacet might also work.